### PR TITLE
Feature of Sharing Link and Statistics Page

### PR DESCRIPTION
**Pull request by ghost**
This pull request fixes two features of the project: Sharing Link and Statistics Page.

### CODE REVIEW

File: frontend/cypress/e2e/features/tests/estatistica.feature [hunk 1]
```diff
@@ -0,0 +1,12 @@
+Feature: Estatísticas
+    As an administrator
+    I want to view various performance metrics
+    So that I can monitor and analyze the system's usage and its profitability.
```
Comment (JDaniielC) [inline frontend/cypress/e2e/features/tests/estatistica.feature:4]: I prefer these phases all in english, but, we made using mix english with portuguese

File: frontend/cypress/e2e/features/tests/estatistica.feature [hunk 2]
```diff
@@ -0,0 +1,12 @@
...
+
+
+Scenario: visualizar estatísticas do restaurante
+Given estou logado com nome "admin"
+And estou na página "Home"
+When acesso a página "Estatistica"
+Then estou na página "Estatistica"
+And consigo ver "totalUsers", "totalitems", "totalOrders", "monthOrders", "totalRevenue", "monthRevenue" e "averageTicket"
```
Comment (JDaniielC) [inline frontend/cypress/e2e/features/tests/estatistica.feature:11]: What are these parameters? It's better identify it.

**Review by JDaniielC — CHANGES_REQUESTED** [review]
Please, do backend features (control and service) for these features.

File: backend/src/stats.json
Comment (JDaniielC) [inline backend/src/stats.json:1]: Delete this file.

File: backend/src/statsApp.ts
Comment (JDaniielC) [inline backend/src/statsApp.ts:1]: Delete this file.

File: backend/tests/features/statistics.feature [hunk 3]
```diff
@@ -0,0 +1,16 @@
+Feature: Statistics
+
+  Scenario: GET data
+    Given I send a GET request to "/stats.json"
```
Comment (JDaniielC) [inline backend/tests/features/statistics.feature]: /stats.json ins't a route pattern, could be only /stats or /statistics

Comment (JDaniielC) [reply backend/tests/features/statistics.feature]: Paramatize the HTTP method, "GET"

File: backend/tests/features/statistics.feature [hunk 4]
```diff
@@ -0,0 +1,16 @@
...
+    When the request is processed
+    Then the response status should be 200
```
Comment (JDaniielC) [inline backend/tests/features/statistics.feature]: Paramatize the HTTP status code "200"

File: backend/tests/features/statistics.feature [hunk 5]
```diff
@@ -0,0 +1,16 @@
...
+    And the response should contain the data object
```
Comment (JDaniielC) [inline backend/tests/features/statistics.feature]: What is "data object" describe it.

File: backend/tests/features/statistics.feature [hunk 6]
```diff
@@ -0,0 +1,16 @@
...
+
+  Scenario: PUT data
+    Given I send a PUT request to "/stats.json" with the following values
+      | totalUsers | 150 |
+      | totalItems | 300 |
+    When the request is processed
+    Then the response status should be 200
+    And the response should contain the updated data
```
Comment (JDaniielC) [inline backend/tests/features/statistics.feature]: Rewrite this scenario.

File: frontend/cypress/e2e/features/tests/estatistica.feature [hunk 7]
```diff
@@ -1,12 +1,11 @@
...
-Given estou logado com nome "admin"
-And estou na página "Home"
-When acesso a página "Estatistica"
-Then estou na página "Estatistica"
-And consigo ver "totalUsers", "totalitems", "totalOrders", "monthOrders", "totalRevenue", "monthRevenue" e "averageTicket"
+Scenario: visualize statistics
+Given I am at page "Home"
+When I access page "Statistics"
+Then I am at page "Statistics"
+And I can see "totalUsers", "totalitems", "totalOrders", "monthOrders", "totalRevenue", "monthRevenue" e "averageTicket"
```
Comment (JDaniielC) [inline frontend/cypress/e2e/features/tests/estatistica.feature]: What are "totalUsers", "totalItems"....

**Review by JDaniielC — CHANGES_REQUESTED** [review]

File: frontend/cypress/e2e/features/tests/estatistica.feature (hunk 7 above)
Comment (ghost) [reply frontend/cypress/e2e/features/tests/estatistica.feature]: They are the statistics data parametrized in attributes from the json object.

File: backend/tests/features/statistics.feature (hunk 6 above)
Comment (ghost) [reply backend/tests/features/statistics.feature]: What should I change?

**Review by JDaniielC — APPROVED** [review]

**Pull request merged**